editor/plugins/spatial_editor_plugin.h

```diff
@@ -203,7 +205,7 @@ class SpatialEditorViewport : public Control {
 
 	struct Cursor {
 
-		Vector3 cursor_pos;
+		//Vector3 cursor_pos; // TODO Lots of dead code associated with this, delete it?
```

Comment:
Sure?
Like, also all code from line 730 to 800 won't do anything. Not sure what it's supposed to do...
https://github.com/godotengine/godot/blob/master/editor/plugins/spatial_editor_plugin.cpp#L730